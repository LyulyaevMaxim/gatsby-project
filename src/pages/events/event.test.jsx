import React from 'react'
import renderer from 'react-test-renderer'
// import Header from './index'

const Header = props => (
  <header>
    <nav>
      <h1>
        <a href="/">{props.siteTitle}</a>
      </h1>
    </nav>
  </header>
)

describe('Header', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Header siteTitle="Default Starter" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
