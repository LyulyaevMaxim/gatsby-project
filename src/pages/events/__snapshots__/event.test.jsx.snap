// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header>
  <nav>
    <h1>
      <a
        href="/"
      >
        Default Starter
      </a>
    </h1>
  </nav>
</header>
`;
